ios/app/AudioMode.h

```diff
@@ -0,0 +1,14 @@
+#import <AVFoundation/AVFoundation.h>
+#import <Foundation/Foundation.h>
+
+#import "RCTBridgeModule.h"
+
+
+@interface AudioMode : NSObject<RCTBridgeModule>
+
+@property (nonatomic, readonly) AVAudioSession *session;
+@property (nonatomic, readonly) NSString *category;
+@property (nonatomic, readonly) NSString *mode;
+@property (nonatomic, readonly) BOOL initialized;
```

Comment:
These properties are not really supposed to be public, right? As far as I understand their use in the respective .m file, they are private.

Further thinking about the .h file brought me to the realization that for the react-native modules that we have in ios/app - AudioMode and POSIX - are not really something that we plan to publicly expose through Objective C. They do have public APIs but these are in JavaScript. So the AudioMode.h and POSIX.h stopped making sense to me.